content/docs/fragments.md

```diff
@@ -66,7 +66,7 @@ results in a `<Table />` output of:
 </table>
 ```
 
-So, we introduce `Fragment`s.
+So, we introduce `Fragments`.
```

Comment:
Let's kill the monospace completely here to avoid confusion

Reply:
I agree! "`Fragments`" isn't valid code in this context, so not having any kind of special formatting is probably better. If we still wanted to have the word stand out, though, we could say: "`Fragment` components". What do you think, @alexkrolick?